efficientdet/third_party/tools/nvgpu.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Python interface for nvidia-smi."""
+
+import subprocess
+from xml.etree import cElementTree as ElementTree
```

Comment:
This file is a branch of nvgpu (it would make it much easier to manage if there are only folders in third_party), but I don't know why it doesn't show the diff to the original file.